packages/my-account/src/assets/locales/en/common.json

```diff
@@ -76,6 +78,39 @@
       "title": "Payments"
     }
   },
+  "subscriptions": {
+    "title": "My subscriptions",
+    "subscriptionContains": "contains {{count}} item",
+    "subscriptionContains_other": "contains {{count}} items"
+  },
+  "noSubscriptions": {
+    "title": "No subscription yet",
+    "description": "Looks like you haven't made any subscriptions yet",
+    "buttonLabel": "Go to shop"
+  },
+  "subscription": {
+    "title": "Subscription #<0/>",
+    "starts_at": "started on <0/>"
+  },
+  "subscriptionFrequency": {
+    "hourly": "hourly",
+    "daily": "daily",
+    "weekly": "weekly",
+    "monthly": "monthly",
+    "two-month": "two-month",

```

Comment:
I would use the same translations as the checkout: https://github.com/commercelayer/mfe-checkout/blob/d193a338f4d197574a8a992fab4adef9d7ae53f4/public/static/locales/en/common.json#L59

Reply:
Sure! Updated them in both locales